frame/staking/src/default_weights.rs

```diff
@@ -96,6 +96,13 @@ impl crate::WeightInfo for () {
 			.saturating_add(DbWeight::get().reads((5 as Weight).saturating_mul(n as Weight)))
 			.saturating_add(DbWeight::get().writes((3 as Weight).saturating_mul(n as Weight)))
 	}
+	fn rebond(l: u32) -> Weight {
+		(71316000 as Weight)
```

Comment:
The same question here.

Reply:
//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 2.0.0

Reply:
I checked the weight values set in the substrate latest code weight and found an inconsistency.

https://github.com/paritytech/substrate/blob/master/frame/staking/src/weights.rs#L362
```rs
fn rebond(l: u32, ) -> Weight {
		(49_966_000 as Weight)
			.saturating_add((92_000 as Weight).saturating_mul(l as Weight))
			.saturating_add(RocksDbWeight::get().reads(4 as Weight))
			.saturating_add(RocksDbWeight::get().writes(3 as Weight))
	}
```
Maybe i missed something here?

Reply:
Every network, every runtime has its own weight. Please learn more about **weight**.

Reply:
https://substrate.dev/docs/en/knowledgebase/learn-substrate/weight

Reply:
`Every network, every runtime, every implementation has its unique own weight` I agree with this view.

According to what you said above, the staking weight `THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 2.0.0-rc6`, and the darwinia common does not seem to have a benchmark for the staking module separately. We have made some changes to the staking module, so the weight settings are inconsistent, it would be better to add some benchmark in the future.

Reply:
Already WIP. If you got some time. Keep going https://github.com/darwinia-network/darwinia-common/tree/xavier-benchmark